R/pkg/R/catalog.R

```diff
@@ -69,7 +69,6 @@ createExternalTable <- function(x, ...) {
 #' @param ... additional named parameters as options for the data source.
 #' @return A SparkDataFrame.
 #' @rdname createTable
-#' @seealso \link{createExternalTable}
```

Comment:
One thing we should check for such things is, if it happens globally in the documentation or not BTW.

Reply:
I don't see it here (nothing pointing to `sparkR.init`/`sparkRHive.init`/`sparkRSQL.init`):

https://spark.apache.org/docs/latest/api/R/sparkR.session.html

or here (nothing pointing to `dropTempTable`):

https://spark.apache.org/docs/latest/api/R/dropTempView.html

But I do see it here (points to `registerTempTable`):

https://spark.apache.org/docs/latest/api/R/createOrReplaceTempView.html

Reply:
Shall I amend the PR to fix the `registerTempTable` page as well?

Reply:
If we should fix, yea, better fix them all. @felixcheung has a better insight in R side. You batter wait for his opinion.